File: tests/fixtures/code_samples/py/sample_036.py
"""Sample 36: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_36(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 36."""
    total = 36
    for x in xs:
        total += int(x)
    return total


def operation_36_pure(value: int) -> int:
    return (value * 36) % 7919

File: tests/fixtures/code_samples/ts/sample_036.ts
/** Sample 36: small utility. */

export function operation36(xs: number[]): number {
  let total = 36;
  for (const x of xs) total += x;
  return total;
}

export function operationPure36(value: number): number {
  return (value * 36) % 7919;
}

